backend/app/blueprints/routes.py

```diff
@@ -185,6 +187,39 @@ def get_single_media(media_id):
     finally:
         session.close()
 
+@BP.route('/search/<term>')
+def search_db(term):
+    """
+    Search database
+    """
+    #select * from search where LOWER(name)
+    # like LOWER('%John%') or LOWER(about) like LOWER('%john%');
+    session = get_session()
+    #query_param_itr = request.args.items()
+    try:
+        #name_param = next(itr).lower()
+        #about_param = next(itr).lower()
+        s = select([search]).\
+                where(
+                    and_(
+                        or_(search.c.name.ilike('%'+term+'%'),
+                            search.c.about.ilike('%'+term+'%'),
+                            search.c.kind.ilike('%'+term+'%'),
+                            search.c.image.ilike('%'+term+'%'),
+                            cast(search.c.id, Text).ilike('%'+term+'%'),
+                            search.c.release_date.ilike('%'+term+'%'))
+                    )
+                )
+        result = session.execute(s).fetchall()
+        count = len(result)
```

Comment:
Count needs to be the total number of possible results, which needs to come from a second query like in the other endpoints